google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -98,19 +143,25 @@ public JComponent createComponent() {
 
   @Override
   public boolean isModified() {
-    return !Comparing.strEqual(getCloudSdkLocation(), configuration.getCloudSdkPath()) ||
-        !Comparing.strEqual(getCloudProjectName(), configuration.getCloudProjectName());
+    boolean isSdkDirModified = !Comparing.strEqual(getCloudSdkDirectory(),
+        CloudSdkUtil.toParentDirectory(configuration.getCloudSdkExecutablePath()));
```

Comment:
Leaving this one as is since the line break alignment that I have matches with the ide formatting method and I don't like force adding spaces/tabs to lines just to get them to line up.